### PR TITLE
Update and rename support-request.md to question.md

### PR DESCRIPTION
**Pull request by hormes**
Signed-off-by: Fansong Zeng <fanster.z@gmail.com>

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+name: Question
+about: Support request or question relating to Koordinator
+title: "[question]"
+labels: kind/question
+
+---
+
+<!-- Please only use this template for submitting question -->
+
+**What happened:**
```
Comment (jasonliu747) [inline .github/ISSUE_TEMPLATE/question.md:11]: this template looks exactly like `bug-report.md`

Comment (hormes) [reply .github/ISSUE_TEMPLATE/question.md:11]: Uses can not feel free to submit bugs

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**